### DISCUSSION TITLE
Nightly NuGet builds

### DISCUSSION THREAD

**Discussion by twsl** [CommunityToolkit/Maui#852] [New Feature Discussions]
For a field evolving as rapidly as MAUI, I'd like to see bugfixes and new features much faster. How about releasing nightly nuget packages to githubs registry?

**Comment by VladislavAntonyuk** [comment]
Feel free to use our public AzureDevops feed: https://dev.azure.com/dotnet/CommunityToolkit/_artifacts/feed/CommunityToolkit-MainLatest

**Reply by VladislavAntonyuk** [reply]
```
<add key="CommunityToolkit-MainLatest" value="https://pkgs.dev.azure.com/dotnet/CommunityToolkit/_packaging/CommunityToolkit-MainLatest/nuget/v3/index.json" />
```

**Reply by twsl** [reply]
Awesome, thats what I have been searching for but couldnt find it linked anywhere in the repo

**Reply by twsl** [reply]
Unfortunately the nuget package version is not set correctly, therefore it causes a lot of problems trying to use it.

**Reply by VladislavAntonyuk** [reply]
What problems do you have?
The version is ordered by build number: https://dev.azure.com/dotnet/CommunityToolkit/_artifacts/feed/CommunityToolkit-MainLatest/NuGet/CommunityToolkit.Maui/versions/1.0.0-preview3578

**Reply by twsl** [reply]
The regular release on nuget is tagged with version 3.1.0, the nightly builds are tagged as 1.0.0-preview. Which means it always shows the regular build as latest version and causes issues with other libraries that require CommunityToolkit >= 3.0.0

**Reply by jfversluis** [reply]
For the versioning it would be cool to read the tagname from this API and deduce the latest major version and up that for the nightly builds: https://api.github.com/repos/CommunityToolkit/Maui/releases

**Reply by twsl** [reply]
Or simply use something like https://github.com/GitTools/GitVersion

**Reply by jfversluis** [reply]
Where is the fun in that 🤡 yeah I actually knew about that one... 🤦 so something like that! Awesome!

**Reply by twsl** [reply]
Should I create an issue and PR for this? Or do you want to fix it?

**Reply by jfversluis** [reply]
If you're willing and able, go ahead!

**Comment by CliffAgius** [comment]
Closed and answered
